### PR TITLE
Top up canister

### PR DESCRIPTION
**Pull request by dprats**
These are changes to make it easier to understand how to top up canisters because there are a few options (3 for dfx, 1 for NNS frontend dapp)

Screenshot 2023-04-28 at 1 46 10 PM

Screenshot 2023-04-28 at 1 46 19 PM

### CODE REVIEW

File: docs/developer-docs/production/topping-up-canister.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+`dfx ledger --network ic top-up --amount 0.1 jqylk-byaaa-aaaal-qbymq-cai`
+
+```bash
+dfx ledger account-id
+dfx ledger --network ic balance
+dfx ledger --network ic top-up --amount 0.1 jqylk-byaaa-aaaal-qbymq-cai
+```
+
+-   The `dfx dfx ledger account-id` returns the `account` id of the current `dfx` identity used
```
Comment (adamspofford-dfinity) [inline docs/developer-docs/production/topping-up-canister.md]:
```suggestion
-   The `dfx ledger account-id` command returns the `account` id of the current `dfx` identity used
```

File: docs/developer-docs/production/topping-up-canister.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+-   The `wallet --network ic balance` checks the cycles balance of your cycles wallet on mainnet
+-   The `canister deposit-cycles` takes cycles from your cycles wallet and gives them to the canister of your choice
+
+
+I want top up canister `jqylk-byaaa-aaaal-qbymq-cai` with a million cycles (1000000).
+
+### Special Case: topping up another cycles wallet
+
+Cycles wallets are canisters like any other, so you can top them up as well. If you have a cycles wallet you control via dfx, there is another option as well for sending cycles from your cycles wallet to a canister of your choice: `dfx wallet send [OPTIONS] <DESTINATION> <AMOUNT>`.
```
Comment (adamspofford-dfinity) [inline docs/developer-docs/production/topping-up-canister.md:89]: The header mentions that it is only used for topping up another cycles wallet, but the body disagrees  ('canister of your choice' would be misleading here)

File: docs/developer-docs/production/topping-up-canister.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+3. There are a few ways to top up canisters so this article goes through each one.
+
+
+## Topping up a canister with `dfx`
+
+In the following examples, we will try to top up canister `jqylk-byaaa-aaaal-qbymq-cai` from the [first tutorial](../../tutorials/deploy_sample_app.md) with a million cycles (1000000). These instructions can work for any canister principal or any cycles amount.
+
+
+### Number of cycles
+For easy of use, you can copy/paste the cycles amounts below:
```
Comment (adamspofford-dfinity) [inline docs/developer-docs/production/topping-up-canister.md:18]:
```suggestion
For ease of use, you can copy/paste the cycles amounts below:
```

File: docs/developer-docs/production/topping-up-canister.md (hunk 2 above)
Comment (dprats) [reply docs/developer-docs/production/topping-up-canister.md:89]: you are right. will fix

File: docs/developer-docs/production/topping-up-canister.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+In this example, we assume we are sending cycles to a cycles wallet with principal `dfds-sddds-aaaal-qbsms-cai`.
+
+```bash
+dfx wallet --network ic balance
+dfx wallet --network ic send 1000000 dfds-sddds-aaaal-qbsms-cai 
+```
+
+-   The `wallet --network ic balance` checks the cycles balance of your cycles wallet on mainnet
+-   The `wallet --network ic send 1000000 dfds-sddds-aaaal-qbsms-cai` takes 1000000 cycles from your cycles wallet and send sthem to cycles wallet `dfds-sddds-aaaal-qbsms-cai`.
```
Comment (adamspofford-dfinity) [inline docs/developer-docs/production/topping-up-canister.md:96]:
```suggestion
-   The `wallet --network ic send 1000000 dfds-sddds-aaaal-qbsms-cai` takes 1000000 cycles from your cycles wallet and sends them to cycles wallet `dfds-sddds-aaaal-qbsms-cai`.
```

File: docs/developer-docs/production/topping-up-canister.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+`dfx ledger --network ic top-up --amount 0.1 jqylk-byaaa-aaaal-qbymq-cai`
+
+```bash
+dfx ledger account-id
+dfx ledger --network ic balance
+dfx ledger --network ic top-up --amount 0.1 jqylk-byaaa-aaaal-qbymq-cai
+```
+
+-   The `dfx ledger account-id` returns the ledger `account` id of the current `dfx` identity used
```
Comment (adamspofford-dfinity) [inline docs/developer-docs/production/topping-up-canister.md:66]:
```suggestion
-   The `dfx ledger account-id` returns the ledger account id of the current `dfx` identity used
```

**Review by dfx-json — APPROVED** [review]

**Pull request merged**